assets/javascripts/kitten/components/typography/text.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+export class Text extends Component {
+  render() {
+    const {
+      align,
+      className,
+      color,
+      decoration,
+      lineHeight,
+      size,
+      style,
+      tag,
+      transform,
+      weight,
+      ...others,
+    } = this.props
+
+    const Tag = tag
+
+    const textClassName = classNames(
+      {
+        // Align.
+        'k-u-align-left': align == 'left',
+        'k-u-align-center': align == 'center',
+        'k-u-align-right': align == 'right',
```

Comment:
On a tranché sur les exemples. Pas sur l'alignement.

La PR reste en `Work in Progress` pour mettre en place le découpage `helper/utilitaire` : 

```js
@mixin k-utilities-size($size: false) {
  @if $size == 'big' {
    @include k-typographyFontSize(1);
  }

  @else if $size == 'default' {
    @include k-typographyFontSize(0);
  }
}

@mixin k-u-size-big {
  .k-u-size-big { @include k-utilities-size('big') }
}
```

Ca va permettre de remplacer les `@extend` de `RichText` par le `helper` en question.